packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -453,9 +454,10 @@ ${summary.join('\n')}
         'out.vmcode',
       );
 
-  String get _debugInfoOutpath => p.join(
+  static const _debugInfoFileName = 'linker_diagnostic.zip';
```

Comment:
I'd maybe give this a more descriptive name – "debug info" could refer to a lot of things. Maybe `_linkDebugInfoFileName`?